circle.yml

```diff
@@ -20,6 +20,9 @@ dependencies:
     # We install and test with all the supported MongoDB versions
     - case $CIRCLE_NODE_INDEX in [1,2]) sudo service mongodb stop ; MONGODB_VERSION=3.2.12 sudo -E .circle/install-and-run-mongodb.sh > /tmp/mongodb-install.log 2>&1 ;; [3,4]) sudo service mongodb stop ; MONGODB_VERSION=3.4.2 sudo -E .circle/install-and-run-mongodb.sh > /tmp/mongodb-install.log 2>&1 ;; esac:
         background: true
+    # We sleep a bit to wait for the background process to start and script to
+    # finish
+    - case $CIRCLE_NODE_INDEX in [1,2]) sleep 10 ;; esac
```

Comment:
It can but I prefer separate lines to make it more readable (flow it's more sequential).

I already hate all the case lines which make it hard to read and follow so I plan to look into circleci-matrix or similar solution in the future so we can at the very least keep circle.yml more sane.